telethon/events/messageread.py

```diff
@@ -29,13 +31,16 @@ async def handler(event):
                 # Log when you read message in a chat (from your "inbox")
                 print('You have read messages until', event.max_id)
     """
-    def __init__(
-            self, chats=None, *, blacklist_chats=False, func=None, inbox=False):
+    def __init__(self,
+                 chats: Optional[Sequence[hints.Entity]] = None, *,
+                 blacklist_chats: bool = False,
+                 func: Optional[Callable[['EventCommon'], None]] = None,
```

Comment:
Shouldn't this use the more specific class?

```suggestion
                 func: Optional[Callable[['MessageRead.Event'], None]] = None,
```